packages/platform-ios/src/commands/runIOS/index.ts

```diff
@@ -398,20 +398,9 @@ function getTargetPaths(buildSettings: string) {
 function getBuildPath(
   xcodeProject: ProjectInfo,
   configuration: string,
-  appName: string,
-  isDevice: boolean,
+  buildOutput: string,
   scheme: string,
 ) {
-  let device;
-
-  if (isDevice) {
-    device = 'iphoneos';
-  } else if (appName.toLowerCase().includes('tvos')) {
-    device = 'appletvsimulator';
-  } else {
-    device = 'iphonesimulator';
-  }
```

Comment:
Side effect, we also support iPad and MacOS as well. 

CC: @alloy, maybe it's time to change `run-ios` to just `run-apple`.

Reply:
(I mean not sure if the command supports these platforms right now, but this is surely first step).